test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1

```diff
@@ -2089,9 +2109,10 @@ Describe "Import-PSSession on Restricted Session" -tags "Feature","RequireAdminO
         }
         else
         {
+            $configName = "restricted_102F"
```

Comment:
If you need it unique, why not use a guid?

Reply:
Fixed